lab-trent/index.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const net = require('net');
+const server = net.createServer();
+
+const clientHandler = require('./lib/clienthandler');
+const Client = clientHandler.Client;
+
+const PORT = 43593;
```

Comment:
Unless you have a compelling reason, I would suggest sticking with the default local host port for working in Node: 3000. 

I'm taking a wild guess that you already have something on that port at home? ;-)